frontend/src/Auth/AuthProvider.js

```diff
@@ -10,7 +10,7 @@ export default function AuthProvider({ children }) {
   const claims = jwt.decode(token)
 
   const user = claims && {
-    username: claims.sub,
+    name: claims.sub,
```

Comment:
überall userName aber hier username ganz weg?

Reply:
Is n Punkt...sollte bei einer Sache bleiben